src/workflow_review.md

```diff
@@ -0,0 +1,128 @@
+
+These steps are described into detail in a [separate workflow](https://inbo.github.io/git-course/workflow_rstudio.html) and in [this manual](https://inbo.github.io/git-course/course_rstudio.html#28_create_logical_commits).
+
+# Create a pull request
+
+In RStudio, make sure all changes are committed and pushed.
+
+In the GitHub repository:
+
+- create a pull request by either
+    - using the `Compare & pull request` button that appears after recent pushes
+    - selecting your new feature branch and hitting the `New pull request` button
+- make sure the correct branches are indicated:
+    - 'base' mentions the branch where will merged to, often the master branch
+    - the 'compare' branch is your feature branch
+- adapt the title if needed, and describe what you did (you can scroll down to check your commits)
+- hit `Create Pull request`
+- request reviewers by adding them on the right top of the page
+
+For an explanation including screenshots, see [here](https://inbo.github.io/git-course/course_rstudio.html#214_pull_requests) or [here](https://inbo.github.io/git-course/workflow_rstudio.html#edits_on_branch_are_ready).
+
+__Remark:__ after creating a pull request, it is still possible to add new
+commits to this branch and pull request.
+After pushing new commits in RStudio, they appear in the pull request.
+To show others that you are not finished yet, you can convert the pull request
+to draft (right top of page, where you request reviewers)
+
+# Revision of code
+
+Reviewers will receive an email with a link to the pull request, and they can:
+
+- inspect
+    - __the changes__ on GitHub in the pull request by inspecting the tabs `Conversation`, `Commits` and `Files changed`
+    - __the adapted version__ locally in RStudio after hitting `Pull` and select the branch name (mentioned at the top of the pull request in GitHub)
+- review it on GitHub by following [this workflow](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-proposed-changes-in-a-pull-request#starting-a-review)
+    - when reviewing, do not mix unrelated items in 1 proposed change, to allow the author to selectively accept changes
+    - to propose major changes for a single issue yourself:
+        - make a new branch in RStudio starting from the branch you are reviewing
+        - add, commit and push your changes
+        - make a pull request to merge your branch to the branch you are reviewing
+
+During this revision, in the `Conversation` tab in the Pull request on GitHub:
+
+- the author can accept suggestions of reviewers by choosing `Commit suggestions`
+- suggestions or comments can be discussed using the comment boxes
+- the author can re-request a reviewer after solving all comments
+
+# Merge changes to master
+
+Once you and your coworkers agreed on the final version in your branch, it can
+be merged to the master branch:
+
+- in RStudio, make sure to include novel changes in the master branch:
```

Comment:
The master branch should be protected. So you can't push changes to master. Hence you must merge to master via GitHub.
In case of conflicts you need to merge the master into the feature branch.

Reply:
Oups, typo, I meant 'from'...

Reply:
solved in 7a2ac2e